docs/content/en/blog/articles/minio-backup/index.md

```diff
@@ -0,0 +1,268 @@
+
+# Deploy K8ssandra
+
+Now that MinIO is up and running, you can create a namespace for your K8ssandra installation and create a secret for Medusa to access the bucket. Create a `medusa_secret.yaml` file with the following content:
+
+```
+apiVersion: v1
+kind: Secret
+metadata:
+ name: medusa-bucket-key
+type: Opaque
+stringData:
+ # Note that this currently has to be set to medusa_s3_credentials!
+ medusa_s3_credentials: |-
+   [default]
+   aws_access_key_id = minio_key
+   aws_secret_access_key = minio_secret
+
+```
+
+Now create the `k8ssandra` namespace and the Medusa secret with the following commands:
+
+```
+kubectl create namespace k8ssandra
+kubectl apply -f medusa_secret.yaml -n k8ssandra
+```
+
+You should now see the **medusa-bucket-key** secret in the **k8ssandra** namespace:
+
+```
+% kubectl get secrets -n k8ssandra
+NAME                  TYPE                                  DATA   AGE
+default-token-twk5w   kubernetes.io/service-account-token   3      4m49s
+medusa-bucket-key     Opaque                                1      45s
+```
+
+You can then deploy K8ssandra with the following custom values file (all default values will be used if not customized here) :
+
+```
+medusa:
+  enabled: true
+  storage: s3_compatible
+  storage_properties:
+      host: minio.minio.svc.cluster.local
+      port: 9000
+      secure: "False"
+  bucketName: k8ssandra-medusa
+  storageSecret: medusa-bucket-key
+```
+
+Save the above file as k8ssandra_medusa_minio.yaml and then install K8ssandra with the following command:
+
+```
+helm install k8ssandra k8ssandra/k8ssandra -f k8ssandra_medusa_minio.yaml -n k8ssandra
+```
+
+Now wait for the Cassandra cluster to be ready by using the following wait command:
+
+```
+kubectl wait --for=condition=Ready cassandradatacenter/dc1 --timeout=900s -n k8ssandra
+```
+
+You should now see a list of pods similar to this:
+
+```
+% kubectl get pods -n k8ssandra
+NAME                                                  READY   STATUS      RESTARTS   AGE
+k8ssandra-cass-operator-547845459-dwg68               1/1     Running     0          6m36s
+k8ssandra-dc1-default-sts-0                           3/3     Running     0          5m56s
+k8ssandra-dc1-stargate-776f88f945-p9twg               0/1     Running     0          6m36s
+k8ssandra-grafana-75b9cb64cc-kndtc                    2/2     Running     0          6m36s
+k8ssandra-kube-prometheus-operator-5bdd97c666-qz5vv   1/1     Running     0          6m36s
+k8ssandra-medusa-operator-d766d5b66-wjt7j             1/1     Running     0          6m36s
+k8ssandra-reaper-5f9bbfc989-j59xk                     1/1     Running     0          2m48s
+k8ssandra-reaper-operator-858cd89bdd-7gfjj            1/1     Running     0          6m36s
+k8ssandra-reaper-schema-4gshj                         0/1     Completed   0          3m3s
+prometheus-k8ssandra-kube-prometheus-prometheus-0     2/2     Running     1          6m32s
+```
+
+# Create some data and back it up
```

Comment:
I'm :-1: on creating a CQL file, copying it to a container, then exec-ing CQLSH.

Alternatives:
1. REST API
2. CQLSH
    1. Run CQLSH to create the schema
    2. Import a CSV file containing the data with `cat data.csv | kubectl exec ... -- cqlsh -e "COPY medusa_test.users FROM STDIN"`
3. CQLSH / Bulk loader - similar to previous item, but with an external bulk-loader and port forwarding to the C* pod

Reply:
Also the number of items here is so small, why not just copy and paste into a CQLSH session?

Reply:
I copy/pasted something from our kubernetes.io documentation 😄 

I'll go the copy/paste to CQLSH approach to keep things simple.

Reply:
Interesting I'll work with the docs folks to get rid of copying into a pod. I thought the last time this came up we resolved it :/